### DISCUSSION TITLE
Is anyone else having issues with the Svelte extra not installing?

### DISCUSSION THREAD

**Discussion by thadkrugman** [LazyVim/LazyVim#3571] [General]
For the past few days, I keep getting an error that Mason is unable to install Svelte and that the typescript-svelte-plugin path can't be found. 

I have tried disabling and re-enabling the extra, changing Node versions on my machine, force installing the package, and clearing the npm cache. 

I am not having issues with Astro, TS, etc.

**Comment by folke** [comment]
You need to explicitely update svelte language server. The warning message you see says exactly this.
So open `:Mason`, go to svelte and press `u`
